packages/pd-console/src/ui/styles/globals.css

```
@@ -4,76 +4,76 @@
 
 @theme {
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "@theme" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)